cores/arduino/zephyrSerial.cpp

```diff
@@ -6,23 +6,35 @@ size_t arduino::ZephyrSerial::begin(unsigned long int baudrate){
     return 0;
 }
 
+size_t arduino::ZephyrSerial::print_char(char ch, bool lf){
+    printk(lf ? "%c\n" : "%c", ch);
+    return lf ? 2 : 1;
+}
+
+size_t arduino::ZephyrSerial::print_str(const char* ptr, bool lf) {
+    printf(lf ? "%s\n" : "%s", ptr);
```

Comment:
```suggestion
   printk(lf ? "%s\n" : "%s", ptr);
```

I have made an error here. This should be `printk`, not `printf`.